tools/pylint-extensions/azure-pylint-guidelines-checker/code_examples.md

```diff
@@ -1163,4 +1164,21 @@ def create_configuration(**kwargs):
     config.logging_policy = NetworkTraceLoggingPolicy(**kwargs)
     config.distributed_tracing_policy = DistributedTracingPolicy(**kwargs)
     return config
+```
+
+## do-not-use-logging-exception
+❌ **Incorrect**:
+```python
+import logging
+
+logging.exception("An error occurred")  # Using logging.exception directly
```

Comment:
should we put this in a try/except with the exception being logged to show representative use?